src/Mount.cpp

```diff
@@ -3429,7 +3430,7 @@ void Mount::moveStepperBy(StepperAxis direction, long steps)
             }
             break;
         case DEC_STEPS:
-            moveSteppersTo(_stepperRA->targetPosition(), _stepperDEC->targetPosition() + steps);
+            moveSteppersTo(_stepperRA->currentPosition(), _stepperDEC->currentPosition() + steps);
```

Comment:
isn't this an ra->stop and dec->move? would be easier to read/understand.

Reply:
moveSteppersBy() is only called by the MX command. and the homing and endswitch code, which both stop the motors before moving them. Using target would be incorrect for them, so current is correct.